tests/rptest/services/redpanda.py

```diff
@@ -1715,6 +1715,19 @@ def pids(self, node):
     def started_nodes(self):
         return self._started
 
+    def render(self, path, **kwargs):
+        # Mitigate a race between checking for template_loader and template_env
+        # inside TemplateRenderer::render
```

Comment:
what is the race? afaict looking at ducktape source this is all single threaded and deterministic.

Reply:
It's because we run start_node in parallel, and each thread is operating on the same RedpandaNode.  I guess maybe the construction of the loader is touching the filesystem or something, to be slow enough for this to happen in practice.

Reply:
>we run start_node in parallel, and each thread is operating on the same RedpandaNode

thanks! i didn't know this, but now that you're mentioning it i'm starting to remember

Reply:
Well, yeah, I don't disagree. It's all I could think of.